modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -1209,6 +1209,12 @@ class NDB_Form_User_Accounts extends NDB_Form
                 $errors['examiner_group'] = "Please specify if examiner is a ".
                     "radiologist";
             }
+            if (preg_match("/^ex_[1-9]+$/", $k)
+                && $values['examiner_radiologist'] !== ''
+                && $values['examiner_pending'] == ''
```

Comment:
I don't understand the logic behind what this code is trying to do or how it relates to the error shown.